app/merginapi.cpp

```diff
@@ -826,7 +826,7 @@ void MerginApi::registrationFinished()
     {
       // the self-registration is not allowed on the server
       emit registrationFailed();
-      emit notify( tr( "New registrations are not allowed on the selected Mergin server.%1Please check with your administrator." ).arg( "<br/>" ) );
+      emit notify( tr( "New registrations are not allowed on the selected Mergin server.\n Please check with your administrator." ).arg( "<br/>" ) );
```

Comment:
These two changes will break the translations... (and the second one has forgotten `.arg(...)`)
How about we just change the arg(...) content to have `\n` inside?